src/utils/api.js

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios'
+import config from './config'
+import { getToken } from './auth'
+
+
+const token = getToken();
+
+const uri = config.BASE_URL;
+
+const headers = {
+  'Content-Type': 'application/json',
+  Authorization: `Token ${token}`,
+};
+
+const api = {
```

Comment:
Instead of creating instances for all the http verbs as below create only one instance that will allow you to access all the verbs below for example
```
const axiosInstance = axios.create({
  baseURL: {{url}},
  headers: {
    Authorization: `Bearer ${token}`,
  },
});
```